src/_js/main.mjs

```diff
@@ -57,3 +74,16 @@ const firstScript = document.scripts[0];
 const scriptElement = document.createElement('script');
 scriptElement.src = 'https://www.google-analytics.com/analytics.js';
 firstScript.parentNode.insertBefore(scriptElement, firstScript);
+
+// Dynamically either insert the dark or the light themed Twitter widget.
+((twitterWidgetOnPage, toggle, prefix, suffix) => {
+  if (twitterWidgetOnPage) {
+    document.querySelector('main').insertAdjacentHTML('beforeend', prefix +
+        (toggle ? toggle.mode : 'light') + suffix);
+  }
+})(
+  document.querySelector('script[src*="platform.twitter.com"]'),
+  document.querySelector('dark-mode-toggle'),
+  '<a href="https://twitter.com/v8js" rel="me nofollow" class="twitter-timeline" data-dnt="true" data-height="1000" data-chrome="noheader nofooter" data-theme="',
+  '">View tweets by @v8js</a>'
+);
```

Comment:
Can we simplify this to something like:

```js
const twitterTimeline = document.querySelector('.twitter-timeline');
if (twitterTimeline) {
  const main = document.querySelector('main');
  const toggle = document.querySelector('dark-mode-toggle');
  const prefix = '<a href="https://twitter.com/v8js" rel="me nofollow" class="twitter-timeline" data-dnt="true" data-height="1000" data-chrome="noheader nofooter" data-theme="';
  const suffix = '">View tweets by @v8js</a>';
  main.insertAdjacentHTML('beforeend', prefix + (toggle ? toggle.mode : 'light') + suffix);
}
```

It’s easier to read, and it also avoids querying the DOM for `dark-mode-toggle` unnecessarily. 

Alternatively, I’d be happy to do it as a follow-up to this patch, no worries.

Reply:
Done, see notes in commit message of https://github.com/v8/v8.dev/pull/254/commits/f8ba3e615c15fed363c0a06b614d103b01595528

Reply:
Awesome! Thanks